python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml

```diff
@@ -177,17 +177,15 @@ keys:
             reed_solomon:
               type: bool
         link_tracking_groups:
-          type: list
-          primary_key: name
-          items:
-            type: dict
-            keys:
-              name:
-                type: str
-                description: Group name.
-              direction:
-                type: str
-                valid_values: ["upstream", "downstream"]
+          type: dict
+          keys:
+            direction:
+              type: str
+              valid_values: ["upstream", "downstream"]
+            names:
+              type: list
+              description: Group names.
+              min_length: 1
```

Comment:
this looks breaking - we won't be able to merge this until AVD 6.0 - lets discuss?